packages/shared/lib/kvstore/KVStore.ts

```diff
@@ -0,0 +1,5 @@
+export interface KVStore {
```

Comment:
could this be organized into an existing directory? Maybe `utils`?

Reply:
Yeah, `db` is fine there. `logger` should probably be moved

Reply:
got it. will move kvstore and lock into utils